json-api-lib.cabal

```diff
@@ -12,6 +12,7 @@ category:            Network
 build-type:          Simple
 maintainer:          Shirren Premaratne
```

Comment:
@gwils would you like to add your name to the list of maintainers?

Reply:
Done, thanks